src/PHPUnit/Controller/AbstractControllerTestCase.php

```diff
@@ -536,40 +539,21 @@ protected function getControllerFullClassName()
 
     /**
      * Assert that the application route match used the given module
-     *
-     * @param string $module
      */
-    public function assertModuleName($module)
+    final public function assertModuleName(string $module): void
```

Comment:
If you change the public API then this is a BC break. Please revert this, otherwise the changes will have to wait until the next main version.